python/adaptivity.py.in

```diff
@@ -333,7 +333,65 @@ def detect_colinearity(mesh, tol = 30):
 
     return bfaces_func
 
-
+def adapt_boundary_regions(original_mesh, adapted_mesh, original_regions):
```

Comment:
Good! I did not know. I will remove the adapt_boundary_regions routine and as soon as I have time I will add a pragmatic_get_boundaryTags call. Thanks!